resources/cassandra-yaml/cassandra/cassandra-yaml-cassandra-3.11.6.edn

```diff
@@ -64,6 +66,12 @@
    :default_value "org.apache.cassandra.auth.CassandraRoleManager"},
   :ssl_storage_port {:type "int", :default_value 7001, :format false},
   :tracetype_repair_ttl {:type "int", :default_value 604800},
+  :disk_optimization_strategy
+  {:type "string",
+   :options
+   [{:label "ssd", :value "ssd"}
+    {:label "spinning", :value "spinning"}],
+   :default_value "ssd"},
```

Comment:
So, generally, when something is commented out in the example cassandra.yaml like so:

https://github.com/apache/cassandra/blob/cassandra-3.11.6/conf/cassandra.yaml#L462-L466

We do not provide a `:default_value` in the definitions. This will cause the config-builder to not output anything for that setting in the cassandra.yaml unless a value is provided explicitly.

When the `:default_value` is specified, then the config-builder will output a value in the cassandra.yaml regardless of whether or not the user provided a value. So we would use that for settings like this:

https://github.com/apache/cassandra/blob/cassandra-3.11.6/conf/cassandra.yaml#L1097

We've encountered cases where this distinction is important as Cassandra sometimes treats "no value provided" differently than "the default value provided explicitly".

Reply:
Okay removing the default value then